src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -211,6 +211,17 @@ export class NativeHostMainService implements INativeHostMainService {
 		}
 	}
 
+	async setWindowMinimumWidth(windowId: number | undefined, minimumWidth: number): Promise<void> {
+		const window = this.windowById(windowId);
+		if (window) {
+			const windowWidth = window.win.getSize()[0];
+			const windowHeight = window.win.getSize()[1];
+			const minWindowHeight = window.win.getMinimumSize()[1];
+			window.win.setMinimumSize(minimumWidth, minWindowHeight);
```

Comment:
I would probably check for both `setMinimumSize` and `setSize` if the values have changed. It could be that this method has side effects in Electron and I think we should not call it if nothing changed.